auslib/web/views/client.py

```diff
@@ -57,37 +57,38 @@ def get(self, **url):
 
         # passing {},None returns empty xml
         if release:
-            response_products = release.getResponseProducts()
             response_blobs = []
-            response_blob_names = release.getResponseBlobs()
-            if response_products:
-                # if we have a SuperBlob of gmp, we process the response products and
-                # concatenate their inner XMLs
-                for product in response_products:
-                    product_query = query.copy()
-                    product_query["product"] = product
-                    response_release, response_update_type = AUS.evaluateRules(product_query)
-                    if not response_release:
-                        continue
-
-                    response_blobs.append({'product_query': product_query,
-                                           'response_release': response_release,
-                                           'response_update_type': response_update_type})
-            elif response_blob_names:
-                for blob_name in response_blob_names:
-                    # if we have a SuperBlob of systemaddons, we process the response products and
+            if release.get('schema_version') == 4000:  # if we have a Superblob
```

Comment:
What's the reasoning behind this special case? This goes against the data driven approach I've been trying to keep us on.